task-log/task_log.go

```diff
@@ -0,0 +1,77 @@
+package taskLog
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/taskcluster/taskcluster-cli/apis"
+	"github.com/taskcluster/taskcluster-cli/extpoints"
+)
+
+type taskLog struct{}
+
+func init() {
+	extpoints.Register("task-log", taskLog{})
+}
+
+func (taskLog) ConfigOptions() map[string]extpoints.ConfigOption {
+	return nil
+}
+
+func (taskLog) Summary() string {
+	return "Outputs the logs for <taskID> as generated, and exits when completes."
+}
+
+func (taskLog) Usage() string {
+	usage := "Usage: taskcluster task-log <taskID>\n"
+	return usage
+}
+
+func (taskLog) Execute(context extpoints.Context) bool {
+	taskID := context.Arguments["<taskID>"].(string)
+
+	// Get route from services.go
+	route := apis.Services["Queue"].BaseURL
```

Comment:
Yeah, I would not do this...

Client for golang has a signed url method: https://godoc.org/github.com/taskcluster/taskcluster-client-go/queue#Queue.GetLatestArtifact_SignedURL
But since we don't need a signed url here.. I would just hardcode the URL pattern...